script.js

```diff
@@ -5,8 +5,22 @@ class BookManager {
     this.titleInput = document.querySelector('#title');
     this.authorInput = document.querySelector('#author');
     this.bookList = document.querySelector('#book-list');
+    this.listContent = document.querySelector('#listContent');
+    this.addNewContent = document.querySelector('#addNewContent');
+    this.contactContent = document.querySelector('#contactContent');
+
     this.addForm.addEventListener('submit', this.handleFormSubmit.bind(this)); // form submit event handler
     window.onload = this.displayBooks.bind(this); // window.onload event handler
+
+    document.querySelector('.item1 a').addEventListener('click', () => {
+      this.showContent(this.listContent);
+    });
+    document.querySelector('.item2 a').addEventListener('click', () => {
+      this.showContent(this.addNewContent);
+    });
+    document.querySelector('.item3 a').addEventListener('click', () => {
+      this.showContent(this.contactContent);
+    });
```

Comment:
- [ ] When I refresh the list of books, it shows the form which should not be, I will encourage you to double-check the logic.
![image](https://github.com/DevKaranJ/AwesomeBooks/assets/86200356/cd6f5937-496c-445d-8701-5ac5154483f6)